base/services/wuauserv/wuauserv.h

```diff
@@ -0,0 +1,11 @@
+#define WIN32_NO_STATUS
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#include <stdarg.h>
+#include <windef.h>
+#include <winbase.h>
+#include <winreg.h>
+#include <winsvc.h>
+
+#define NDEBUG
+#include <debug.h>
```

Comment:
Please end with a newline.
(In case you are wondering about the reason for this: https://stackoverflow.com/a/72409 )

Reply:
Thanks for link!